ush/job_preamble.sh

```diff
@@ -42,10 +42,11 @@ export SENDWEB="${SENDWEB:-${SENDWEB_dfv}}"
 export KEEPDATA="${KEEPDATA:-${KEEPDATA_dfv}}"
 export MAILTO="${MAILTO:-${MAILTO_dfv}}"
 export MAILCC="${MAILCC:-${MAILCC_dfv}}"
+export PDYm1=$( $DATE_UTIL --date "${PDY} -1 day" "+%Y%m%d" )
 
 if [ "${RUN_ENVIR}" = "nco" ]; then
     [[ "$WORKFLOW_MANAGER" = "rocoto" ]] && export COMROOT=$COMROOT
-    export COMIN="${COMIN:-$(compath.py -o ${NET}/${model_ver}/${RUN}.${PDY}/${cyc})}"
+    export COMIN="${COMIN:-$(compath.py -o ${NET}/${model_ver}/${RUN}.${PDY})}"
```

Comment:
@JianpingHuang-NOAA, why are you removing `cyc` from `COMIN` and add it separately in the ex-scripts?

Reply:
This is something that we have to do with the AQM since it needs a flexible cyc with COMIN of COMINm2 which can be used for handling chemical restart files for varying cycles.

Reply:
Got it. I am testing your PR now.